proofs/libsectorbuilder/cgo_transforms.go

```diff
@@ -81,9 +81,14 @@ func goPieceMetadata(src *C.sector_builder_ffi_FFIPieceMetadata, size C.size_t)
 
 	ptrs := (*[1 << 30]C.sector_builder_ffi_FFIPieceMetadata)(unsafe.Pointer(src))[:size:size]
```

Comment:
nit: not this PR but this magic number could use some explaining for the uninitiated.